### ISSUE TITLE
`exclude` options

### ISSUE THREAD

**Issue by artola** [angelozerr/tslint-language-service#79]
TLS is not respecting the `tsconfig.exclude` option.

Could be this possible fixed in TLS or is it a tslint issue?

Note: tslint CLI respects `linterOptions.exclude`

**Comment by artola** [comment]
@angelozerr do you have some hint regarding `linterOptions .exclude` ?

I have this `tslint.json` for example:

```json
{
  "extends": "tslint:recomended",
  "linterOptions": {
    "exclude": ["src/**/*"]
  }
}
```
That should not show any error in the whole `src` folder, while VScode/TLS still report them.
Same config, but running `tslint` CLI, show no errors as expected.
